stratuslab/python/tar/slipstream_stratuslab/StratusLabIterClientCloud.py

```diff
@@ -93,6 +92,19 @@ def _start_image_for_deployment(self, node_instance, vm_name):
             raise
         return runner
 
+    def _ensure_pdisk_endpoint_is_set(self):
```

Comment:
I think we should try to set `pdiskEndpoint` from the environment variable in any case because it can have a default value in the SL ConfigHolder.

So I would do something like:
```python
slipstream_pdisk_endpoint = os.environ.get('SLIPSTREAM_PDISK_ENDPOINT')
if slipstream_pdisk_endpoint:
    self.slConfigHolder.set('pdiskEndpoint', slipstream_pdisk_endpoint)
if not self.slConfigHolder.pdiskEndpoint:
    raise Exceptions.ExecutionException('PDisk endpoint should be set on StratusLab ConfigHolder')
```
Here, `SLIPSTREAM_PDISK_ENDPOINT` takes precedence of the value that may have been set in `self.slConfigHolder.pdiskEndpoint`. 
@konstan : Please check that this behaviour is ok.